theano/tensor/nnet/nnet.py

```diff
@@ -384,6 +384,7 @@ def c_code(self, node, name, inp, out, sub):
             }
         }
 
+        #pragma omp parallel for schedule(static, 32)
```

Comment:
Hi nouiz,

I have commited the code to split the small size input for Softmax and SoftmaxGrad. Please help review the code.

Thanks a lot.